web/init-letsencrypt.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
```

Comment:
This file actually lives on the server. It does the initial setup of the certificate, which I've already done. It is here for reference, but unless things go wrong, we shouldn't need to run it again.